.circleci/config.yml

```diff
@@ -104,6 +105,14 @@ jobs:
       - upload_logs:
           file: test-stats
       - notify_slack
+      - store_artifacts:
```

Comment:
You could extract this into a command, like:
```
  upload_gas_report:
    description: 'Upload gas report '
    parameters:
      package:
        type: string
    steps:
      - store_artifacts:
          path: /home/circleci/packages/<< parameters.package >>./gas.json
          destination: << parameters.package >>-gas.json
      - store_artifacts:
          path: /home/circleci/packages/<< parameters.package >>./gas.txt
          destination: << parameters.package >>-gas.txt
```

then, this could be:
```
- upload_gas_report:
  package: rps
- upload_gas_report:
  package: nitro-protocol
```
making it a little easier to maintain